App/Logic/ExposureDetectionLogic.swift

```diff
@@ -59,11 +59,57 @@ extension Logic.ExposureDetection {
         isUserCovidPositive: state.user.covidStatus.isCovidPositive,
         forceRun: self.forceRun
       ))
-
+        
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/ExposureDetectionLogic.swift;
  Line: 62;
-->

- :warning: swiftlint (0.39.2): Lines should not have trailing whitespace. (trailing_whitespace)